Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.UWP/Package.StoreAssociation.xml

```diff
@@ -2,6 +2,7 @@
 <StoreAssociation xmlns="http://schemas.microsoft.com/appx/2010/storeassociation">
   <Publisher>CN=B2D1C358-6AF8-4416-BF73-129CC1F3C152</Publisher>
   <PublisherDisplayName>Azure Mobile Team</PublisherDisplayName>
+  <DeveloperAccountType>MSA</DeveloperAccountType>
```

Comment:
I'm still seeing changes to this file in the PR.
I thought we talked about reverting those changes.

Reply:
There are 2 files (forms demo, and regular demo) that needs to be reverted so that we don't see any difference in the PR.

Reply:
Sorry, I changed wrong file last time.